.cirrus/modules/qa.star

```diff
@@ -134,7 +124,7 @@ def qa_ruling_script():
     "git submodule update --init --depth 1",
     "source cirrus-env QA",
     "source .cirrus/use-gradle-wrapper.sh",
-    "./gradlew \"${GRADLE_TASK}\" \"-Dsonar.runtimeVersion=${SQ_VERSION}\" --tests \"PhpGeneralRulingTest.test${PHP_PROJECT}\" --info --build-cache --console plain --no-daemon"
+    "./gradlew \"${GRADLE_TASK}\" \"-Dsonar.runtimeVersion=${SQ_VERSION}\" --info --build-cache --console plain --no-daemon"
```

Comment:
Instead of having n tasks for each integration test, I put everything into one. Running the CI takes a bit more time overall (around a minute), but it seems better than not running some tests (for now, at least). Four classes were never run.